plugins/window_size/lib/src/window_size_channel.dart

```diff
@@ -26,31 +26,60 @@ const String _windowSizeChannelName = 'flutter/windowsize';
 ///
 /// Returns a list of screen info maps; see keys below.
 const String _getScreenListMethod = 'getScreenList';
+
 /// The method name to request information about the window containing the
 /// Flutter instance.
 ///
 /// Returns a list of window info maps; see keys below.
 const String _getWindowInfoMethod = 'getWindowInfo';
+
 /// The method name to set the frame of a window.
 ///
 /// Takes a frame array, as documented for the value of _frameKey.
 const String _setWindowFrameMethod = 'setWindowFrame';
 
+/// The method name to set the minimum size of a window.
+///
+/// Takes a window size array, as documented for the value of _windowSizeKey.
+const String _setWindowMimumumSizeMethod = 'setWindowMinimumSize';
```

Comment:
Should be `_setWindowMinimumSizeMethod`, not `_setWindowMimumumSizeMethod`, no? Same for `_getWindowMimumumSizeMethod` (as well as references to both).

Reply:
Fixed, thanks for the review =)

Reply:
Agreed, and fixed. Thanks!